include/igl/tinyply.cpp

```diff
@@ -0,0 +1,720 @@
+#include "tinyply.h"
+
+////////////////////////////////
+//   tinyply implementation   //
+////////////////////////////////
+
+#include <algorithm>
+#include <functional>
+#include <type_traits>
+#include <iostream>
+#include <cstring>
+
+namespace igl::tinyply {
```

Comment:
I think this form of nested namespace is only available in C++17.
Also, there should be no need for a `using namespace igl::tinyply` just below.